### DISCUSSION TITLE
Sketch input in the DeepFillV2 model

### DISCUSSION THREAD

**Discussion by ankuPRK** [open-mmlab/mmediting#1219] [Models] [answered]
Hey guys thanks so much for opening up and implementing so many models for opensource community.

I wanted to ask if the DeepFillV2 model in mmediting also supports user-sketch input, like in the original paper:

![image](https://user-images.githubusercontent.com/13609663/128582189-4917ff8c-2354-48a8-bfda-6f3f36d24206.png)

**Comment by zengyh1900** [comment, marked as answer]
Hi @ankuPRK 

Yes. The DeepFillv2 model in MMEditing supports deals with input images with sketch input. 
You can check the source code here: https://github.com/open-mmlab/mmediting/blob/master/mmedit/models/backbones/encoder_decoders/encoders/deepfill_encoder.py#L10 

Unfortunately, now, MMEditing only provides the pre-trained DeepFillv2 model that deals with input images without sketch input. 

Don't hesitate to contribute this new feature/model to MMEditing!

**Reply by ankuPRK** [reply]
Hi @zengyh1900 , thanks for the clarification.
